index.html

```diff
@@ -1,100 +1,122 @@
-    <option value="20">Okreg wyborczy 20</option>
-    <option value="21">Okreg wyborczy 21</option>
-    <option value="22">Okreg wyborczy 22</option>
-    <option value="23">Okreg wyborczy 23</option>
-    <option value="24">Okreg wyborczy 24</option>
-    <option value="25">Okreg wyborczy 25</option>
-    <option value="26">Okreg wyborczy 26</option>
-    <option value="27">Okreg wyborczy 27</option>
-    <option value="28">Okreg wyborczy 28</option>
-    <option value="29">Okreg wyborczy 29</option>
-    <option value="30">Okreg wyborczy 30</option>
-    <option value="31">Okreg wyborczy 31</option>
-    <option value="32">Okreg wyborczy 32</option>
-    <option value="33">Okreg wyborczy 33</option>
-    <option value="34">Okreg wyborczy 34</option>
-    <option value="35">Okreg wyborczy 35</option>
-    <option value="36">Okreg wyborczy 36</option>
-    <option value="37">Okreg wyborczy 37</option>
-    <option value="38">Okreg wyborczy 38</option>
-    <option value="39">Okreg wyborczy 39</option>
-    <option value="40">Okreg wyborczy 40</option>
-    <option value="41">Okreg wyborczy 41</option>
-  </select>
-  <div id="wybranyokreg"></div>
-
-
-
-
-
- <!-- skrypt do odczytu plikow z excela znaleziony w necie -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/xlsx/0.17.2/xlsx.full.min.js"></script>
-
-
-<script>
-document.getElementById('przyciskexcel').addEventListener('click', function() {
-document.getElementById('filei').click();
-});
-
-document.getElementById('filei').addEventListener('change', function() {
-  var file = this.files[0];
-  if (file) {
-  loadedFile = file;
-  alert('Plik Excel został załadowany i zapisany.');
-      }
-    });
-    
-//tutaj znajduje sie kod do wczytania danych z excela 
-    document.getElementById('select').addEventListener('change', function() {
-      var selectedOption = parseInt(this.value);
-      if (loadedFile) {
-        var reader = new FileReader();
-        reader.onload = function(e) {
-          var data = new Uint8Array(e.target.result);
-          var workbook = XLSX.read(data, { type: 'array' });
-          var worksheet = workbook.Sheets[workbook.SheetNames[0]];
-          var selectedCell = worksheet[XLSX.utils.encode_cell({ r: selectedOption, c: 5 })];
-          var cellValue = selectedCell ? selectedCell.v : 'Brak danych';
-          document.getElementById('wybranyokreg').textContent = "Wybrany okreg wyborczy zawiera " + cellValue + " wyborcow";
-        };
-
-        reader.readAsArrayBuffer(loadedFile);
-      } else {
-        alert('Proszę najpierw załadować plik Excel.');
-      }
-    });
-  </script>
+    <h1>liczba wyborcow w danych okregach wyborczych</h1>
+    <button id="przyciskcsv">Załaduj plik CSV</button> <br>
+    <input type="file" id="filei" accept=".csv" style="display: none;">
+    <select id="select"></select>
+    <div id="wybranyokreg"></div>
+    <div id="kwadratodleglosci"></div>
+
+    <script>
+        var loadedFile;

```

Comment:
`var` variable declarations are pretty old and not used anymore. In general, `let` or `const` are more widespread. I suggest using `let` and `const` instead of `var`.

`var` has some weird behavior, such as:

* they are scoped (accessible) in the function that they are declared in. Normally, in other programming languages, variables are block-scoped - only accessible in the block (`{ ... }`) they are defined in.
* top-level `var` variables become global variables. Notice that e.g. `loadedFile` becomes a global variable now and is also accessible through `window.loadedFile`

    
    <img width="196" alt="image" src="https://github.com/Kubast06/Praktyki/assets/889383/527216e3-823e-40a0-a175-b58abb29d3ae">

    This is a screenshot from the Firefox Developer Tools. You can see how `window.addedDistricts` is defined as `[]`, so it accidentally became a global variable.

    Global variables should be avoided, especially if they are accidental 😄 

Using `const` and `let` does not have the same effects, and is generally much safer.

I suggest using `const` unless you need to reassign the variable, in which case use `let`.

Feel free to go through the documentation for both of these declaration types:
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

If you have questions, we can discuss them on a call

Reply:
Okay, thanks, I'll see what you send.